src/products/db.ts

```diff
@@ -0,0 +1,16 @@
+import {Sequelize} from "sequelize";
+
+export const sequelize = new Sequelize({
+    dialect: 'sqlite',
+    storage: './src/products/database.sqlite'
+});
+
+export const connectDB = async () => {
+    try {
+        await sequelize.sync();
+        await sequelize.authenticate();
+        console.log("Connected to database");
+    } catch (e) {
+        console.error("Failed to connect to database", e);
+    }
+};
```

Comment:
this great ! quite similar to what my goal was.
```suggestion
import {Sequelize} from "sequelize";

const productsDBConnection = new Sequelize({
    dialect: 'sqlite',
    storage: './src/products/database.sqlite'
});

export const  productsDB = {
  $connect: async () => {
    try {
        await productsDBConnection.sync();
    } catch (e) {
        console.error("Failed to connect to database", e);
    }
};
$close: async () => {
  await productsDBConnection.close();
  console.info("Closing connection to products database.");
},
connection: productsDBConnection
}
```

Reply:
Could you please explain what is the reasoning  behind adding "$" prefix before the close and connect methods?

Reply:
oh. no real reasoning, just something i picked up along the way to differentiate the db methods from mine. no real reason.